docs/libs/qrcode.rst

```diff
@@ -11,12 +11,12 @@ Usage
 
 Enable :c:macro:`LV_USE_QRCODE` in ``lv_conf.h``.
 
-Use :c:expr:`lv_qrcode_create()` to create a qrcode object, and use
-:c:expr:`lv_qrcode_update()` to generate a QR code.
+Use :cpp:expr:`lv_qrcode_create()` to create a qrcode object, and use
```

Comment:
this should be :cpp:func:\`lv_qrcode_create\`

Reply:
fixed